src/main.rs

```diff
@@ -1,30 +1,53 @@
 use bevy::prelude::*;
```

Comment:
Maybe it is time to split this file a little, it is not good that all the code is in the same file